gcs_oauth2_boto_plugin/oauth2_client.py

```diff
@@ -438,7 +470,7 @@ def GetCredentials(self):
 # be refactored into oauth2client directly in a way that allows for setting of
 # user agent and scopes. https://github.com/google/oauth2client/issues/164
 # pylint: disable=protected-access
-class ServiceAccountCredentials(service_account._ServiceAccountCredentials):
+class ServiceAccountCredentials(ServiceAccountCredentials):
```

Comment:
In the case of oauth2client >=2.0.0, I think we should rely on their existing classes (which support from_json, to_json).